### PR TITLE
🌸✨ `Members`: `Invitation` email conveys which `Space` and `Person` it came from

### PR DESCRIPTION
**Pull request by KellyAH**
- for https://github.com/zinc-collective/convene/issues/1481
---
# Changes
- modify invitation email `From`, `Subject`, `Body` text to more clearly communicate the invitor's name and the `Space` the invitee is being asked to join.

before

**After**
Screenshot 2023-05-21 at 8 20 59 PM

### CODE REVIEW

**Comment by KellyAH** [comment]
IMHO adding the invitor name to the Subject could be too wordy. What do ya'll think? @anaulin @zspencer 

too wordy maybe

**Comment by anaulin** [comment]
What about putting the invite name in the "from" field I stead?

It says "Neighborhood support", but maybe it could say something like "INVITOR NAME via NEIGHBORHOOD". Or some other wording that you prefer. The main point is to convey both the invite name somewhere in the "from" or the "subject".

For inspiration, here are some screencaps of invites from other systems that I found in my inbox:

**Review by zspencer — APPROVED** [review]
Doh, I think I should have left most of my comments from the other PR on this one! Apologies!

I would prefer if we:
- Did not add an additional parameter to the `SpaceInvitationMailer`
- Kept the messaging focused on our client's Space, rather than on Convene

But I'm available for you to merge this as is if you want to get it nailed down.

**Comment by KellyAH** [comment]
Let me rework this on Sunday from @anaulin 's and @zspencer 's suggestions, feedback and requirements. I think I can meet everyone's needs.

**Comment by KellyAH** [comment]
My bad 🤦‍♀️ . I confused the `From` field for the email ending salutation. Thanks for the examples @anaulin  they helped a lot.

Ok I fixed it up. Hope it's better now!

**Comment by KellyAH** [comment]
🤔 Darn browser tests in `responding_to_invitations.feature` is failing in parts of code I don't think I touched.

`AssertionError [ERR_ASSERTION]: By(css selector, *[aria-label="Configure Space"])`

`Couldn't find email {"to":"katara-9d14ec59-5507-4416-8e8d-74ce32c3c734@example.com"}`

Can't see the browser when tests run in code spaces. 😞  I'll need to spin up local again.....

**Comment by zspencer** [comment]
Sometimes the browser specs flake, If you re-run them they often pass. I've kicked off a re-run to see if that's the case here.

File: app/views/space_invitation_mailer/space_invitation_email.text.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Hi <%= @invitation.name %>,
 
-You've been invited to <%= @invitation.space.name %> by <%= @invitation.invitor_display_name %>.
+<%= @invitation.invitor_display_name %> has invited you to join the <%= @invitation.space.name %> Space.
```
Comment (zspencer) [inline app/views/space_invitation_mailer/space_invitation_email.text.erb:3]: Let's ditch the word "Space" from this; since the space name should stand alone.

**Review by zspencer — APPROVED** [review]
Thank you this is a great improvement!

It may be cool (but not particularly urgent) to set up something where we allow the Space to customize the template email copy using something like [Liquid](https://shopify.github.io/liquid/)

**Comment by zspencer** [comment]
OK, with the re-run I'm guessing it's because of how we're searching the sent mail in the test that's failing. I'll take a stab at getting it fixed and push a PR this-a-way.

File: app/views/space_invitation_mailer/space_invitation_email.text.erb (hunk 1 above)
Comment (KellyAH) [reply app/views/space_invitation_mailer/space_invitation_email.text.erb]: fixed in a3ecf1e1bc40279f3745269e09862ab73f1ccad7

Comment (KellyAH) [reply app/views/space_invitation_mailer/space_invitation_email.text.erb]: And removed "the" so it sounds better.
Screenshot 2023-05-22 at 9 03 05 PM

**Comment by zspencer** [comment]
I think this is good to go! We could probably do a follow up PR to try and make the output better when the system tests' email query files (like, why does it just output the "to" when there is also a "text" matcher?! And how do we get a better stack trace?!) but I didn't want to get nerd-sniped by that until I wrap up the tests for fixing the quantitys.
